doc/source/drivers/vector/gml.rst

```diff
@@ -644,6 +644,14 @@ The following open options are supported:
       Whether to use schema imports in XSD files so that
       the feature types corresponding to imported schema can be detected.
 
+-  .. oo:: OGR_SCHEMA
+      :choices: <filename>|<json string>
+      :since: 3.9.4
+
+      Partially or totally overrides the auto-detected schema to use for creating the layer.
+      The overrides are defined as a JSON list of field definitions.
+      This can be a filename or a JSON string or a URL.
```

Comment:
```suggestion
      This can be a filename or a JSON string or a URL conformant with the `ogr_fields_override.schema.json schema <https://raw.githubusercontent.com/OSGeo/gdal/refs/heads/master/ogr/data/ogr_fields_override.schema.json>`__
```